jena-core/src/main/java/jena/rdfcompare.java

```diff
@@ -108,11 +105,6 @@ protected static void usage() {
     
     protected static void read(Model model, String in, String lang) 
       throws java.io.FileNotFoundException {
-        try {
-            URL url = new URL(in);
-            model.read(in, lang);
-        } catch (java.net.MalformedURLException e) {
-            model.read(new FileInputStream(in), "", lang);
-        }
+        model.read(in, lang);
```

Comment:
An historical note: 

That is a way to handle URLs and plain file names.  It only works as plain `model.read(in, lang);` if RIOT is around otherwise there is no HTTP processing of `model.read(String, .....`